va/va_backend.h

```diff
@@ -133,6 +133,30 @@ struct VADriverVTable
 		VAContextID context
 	);
 
+	VAStatus (*vaCreateMFContext) (
+		VADriverContextP ctx,
+		VAMFContextID *mfe_context    /* out */
+	);
+
+	VAStatus (*vaMFAddContext) (
+		VADriverContextP ctx,
+		VAMFContextID mf_context,
+		VAContextID context
+	);
+
+	VAStatus (*vaMFReleaseContext) (
+		VADriverContextP ctx,
+		VAMFContextID mf_context,
+		VAContextID context
+	);
+
+	VAStatus (*vaMFSubmit) (
+		VADriverContextP ctx,
+		VAMFContextID mf_context,
+		VAContextID *contexts,
+		int num_contexts
+	);
+
 	VAStatus (*vaCreateBuffer) (
```

Comment:
Please add the new callback function pointers after the existing callback functions and reduce the reserved bytes. otherwise it will break compatibility between libva and driver.

Reply:
Done